ui/components/multichain/account-overview/account-overview.tsx

```diff
@@ -33,8 +38,11 @@ export function AccountOverview(props: AccountOverviewProps) {
           title={t('basicConfigurationBannerTitle')}
         />
       )}
-      {isEth && <AccountOverviewEth {...props} />}
-      {isUnknown && <AccountOverviewUnknown {...props} />}
+      {isEth && <AccountOverviewEth {...props}></AccountOverviewEth>}
+      {isBtc && <AccountOverviewBtc {...props}></AccountOverviewBtc>}
+      {isUnknown && (
+        <AccountOverviewUnknown {...props}></AccountOverviewUnknown>
+      )}
```

Comment:
Nice to have: We should probably make them exclusive between each other, like a switch-case logic, just to avoid rendering two overview because of a selector bug

Reply:
This is a great idea! I could make a function for this, I'll do that!